host/4/bullseye/amd64/python/python310/python310-buildenv.Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN apt-get update && \
     locale-gen && \
     # install MS SQL related packages
     apt-get update && \
-    apt-get install -y unixodbc msodbcsql17 mssql-tools && \
+    apt-get install -y unixodbc msodbcsql17=17.10.4.1-1 mssql-tools && \
```

Comment:
Can we add a comment here or at L31 with the issue PR number for reference?

Reply:
Updated. Thanks